docs/getting-started/archive-on-dockstore.rst

```diff
@@ -0,0 +1,68 @@
+Archiving an Entry on Dockstore
+===================================
+
+Tutorial Goals
+--------------
+
+-  Archiving an entry
+-  Unarchiving an entry
+
+This tutorial walks through the process of archiving entries from Dockstore.
+
+This action will make your entry read-only and less prominent in the Dockstore user interface.
+You can unarchive your entry back to normal at any time.
+
+
+Specific Changes When Archived
+-------------------------------
+
+After you archive the Dockstore workflow, it will be:
+
+- Marked as "Archived".
+- Read-only, except to unarchive. GitHub updates will be ignored.
+- Ranked lower in Dockstore Search.
+- Displayed in the "Archived" section of your Dockstore dashboard.
+
+Archiving an Entry
+-------------------
+
+Navigate to ``/my-tools``, ``/my-workflows``, ``/my-notebooks``, or ``/my-services`` by going to My Dashboard then selecting the desired option in the left sidebar.
+In the sidebar accordion, find the GitHub repository that your entry is in and click the entry you would like to archive.
```

Comment:
I don't think Archiving is a GitHub-only thing; it works for hosted workflows, and should work for Gitlab and Bitbucket as well.